### PR TITLE
[Cache] allow/provide psr/simple-cache v2

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

As we already did for psr/cache.
See https://github.com/php-fig/fig-standards/pull/1252 for background.

### CODE REVIEW

File: src/Symfony/Component/Cache/composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     ],
     "provide": {
         "psr/cache-implementation": "1.0|2.0",
-        "psr/simple-cache-implementation": "1.0",
+        "psr/simple-cache-implementation": "1.0|2.0",
```
Comment (derrabus) [inline src/Symfony/Component/Cache/composer.json:20]: This should be done in the root composer.json as well, shouldn't it?

**Review by derrabus — CHANGES_REQUESTED** [review]
There is no v2 yet. Shall we wait until we can actually test against the new interfaces?

Bildschirmfoto 2021-09-20 um 13 11 45

File: src/Symfony/Component/Cache/composer.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
         "doctrine/cache": "^1.6|^2.0",
         "doctrine/dbal": "^2.7|^3.0",
         "predis/predis": "^1.1",
-        "psr/simple-cache": "^1.0",
+        "psr/simple-cache": "^1.0|^2.0",
```
Comment (derrabus) [inline src/Symfony/Component/Cache/composer.json:38]: This should be done in the root composer.json as well, shouldn't it?

**Comment by stof** [comment]
@derrabus the work is in progress to create them.

File: src/Symfony/Component/Cache/composer.json (hunk 2 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/Cache/composer.json:38]: right, updated

**Comment by stof** [comment]
@nicolas-grekas I suggest we avoid merging this PR until they merge the work on the FIG side, for safety.

**Comment by interlab** [comment]
> Error: Interface 'Psr\SimpleCache\CacheInterface' not found

symfony/cache (v5.3.8)

```php
$cache = new Psr16Cache(new FilesystemAdapter('', 0, CACHE_DIR));
```


Solution:
`composer require psr/simple-cache`

**Comment by nicolas-grekas** [comment]
psr/simple-cache v2&3 now released, PR rebased and ready

**Pull request merged**